CONTRIBUTING.md

```diff
@@ -62,6 +62,47 @@ Config dir: /usr/local/etc/jupyter
 
 Then start Jupyter Notebook with:
 
+```bash
+jupyter notebook
+```
+### Alternative : Using "yarn" and "npm" over "mamba"
```

Comment:
Even with `mamba`, we still need to use `yarn` or `npm` to work with the JavaScript dependencies.

So maybe there could be a mention to creating environment `venv` after the `mamba` instructions?